src/main/scala/tech/cryptonomic/conseil/Conseil.scala

```diff
@@ -39,27 +41,32 @@ object Conseil extends App with LazyLogging with EnableCORSDirectives with Conse
         enableCORS {
           validateApiKey { _ =>
             logRequest("Conseil", Logging.DebugLevel) {
-              pathPrefix("tezos") {
-                tezos.route
-              } ~ pathPrefix("info") {
-                AppInfo.route
-              }
-            } ~ pathPrefix("v2") {
-              logRequest("Metadata Route", Logging.DebugLevel) {
-                pathPrefix("metadata") {
-                  platformDiscovery.route
-                }
-              } ~ logRequest("Data Route", Logging.DebugLevel) {
-                pathPrefix("data") {
-                  data.getRoute ~ data.postRoute
-                }
-              }
+              tezos.route ~
+              AppInfo.route
+            } ~
+            logRequest("Metadata Route", Logging.DebugLevel) {
+              platformDiscovery.route
+            } ~
+            logRequest("Data Route", Logging.DebugLevel) {
+              data.getRoutes ~ data.postRoute
```

Comment:
`data.getRoute`? for more consistency... or is it already an api call?

Reply:
ok, renamed it